docker-registry/templates/tests/test-pod.yaml

```diff
@@ -1,21 +1,21 @@
 apiVersion: v1
 kind: Pod
 metadata:
-  name: {{ .Chart.Name }}-test
+  name: {{ template "docker-registry.fullname" . }}-test
   labels:
-    app: {{ .Chart.Name }}-test
+    app: {{ template "docker-registry.fullname" . }}-test
   annotations:
     "helm.sh/hook": test-success
 spec:
   containers:
-    - name: {{ .Chart.Name }}-test
+    - name: {{ template "docker-registry.fullname" . }}-test
       image: curlimages/curl
       command:
         - /bin/sh
         - -c
         - |
-          sleep 5
-          curl -I http://docker-registry.$(NAMESPACE).svc.cluster.local:5000
+          sleep 30
+          curl -I http://{{ tpl .Values.test.url . }}:{{ .Values.service.port }}
```

Comment:
@cw-sakamoto 
I think it is better to check in a loop.
Do you have a reason to be `sleep`?

Reply:
@cw-ozaki 
I tried to do that at first(https://github.com/chatwork/charts/pull/246/commits/5f011ea487a8b1aa6297f447cbd39394f8dfa324 ), but I couldn't resolve the name in initContainer, so I gave up.....